docs/flashbots-data/mev-inspect-py/contributing.md

```diff
@@ -0,0 +1,162 @@
+---
+title: contributing
+---
+
+### Tracing
+
+While simple ETH and token transfers are trivial to parse/filter (by processing their transaction input data, events and/or receipts), contract interactions can be complex to identify. EVM tracing allows us to dig deeper into the transaction execution cycle to look through the internal calls and any other additional proxy contracts the tx interacts with, this is useful for the comprehensive analysis we're interested in.
+
+Trace types (by `action_type`):
+
+* `Call`, which is returned when a method on a contract (same as the tx `to` field or a different one within) is executed. We can identify the input parameters in each instance by looking at this sub trace. 
+* `Self-destruct`, when a contract destroys the code at its address and transfers the ETH held in the contract to an EOA. Common pattern among arbitrage bots given the gas refund savings. 
+* `Create`, when a contract deploys another contract and transfers assets to it. 
+* `Reward`, pertaining to the block reward and uncle reward, not relevant here. 
+
+Note that this is for Erigon/OpenEthereum `trace` module and Geth has a different tracing mechanism that is more low-level/irrelevant for inspect.
+
+### Creating an inspector from scratch
+
+If you intend to create your own inspector and submit it as a PR, this should serve as a useful walkthrough to understand the code structure and types. 
```

Comment:
People aren't contributing inspectors anymore, no? The idea was to have people contribute trace classifiers and have strategy inspectors be more general if I recall correctly